src/spec-node/dockerCompose.ts

```diff
@@ -118,10 +120,19 @@ function getInfoForService(composeService: any) {
 			image
 		};
 	}
+	if (typeof (composeBuild) === 'string') {
+		return {
+			image,
+			build: {
+				context: composeBuild,
+				dockerfilePath: 'Dockerfile'
+			}
+		};
+	}
 	return {
 		image,
 		build: {
-			dockerfilePath: composeBuild.dockerfile as string,
+			dockerfilePath: (composeBuild.dockerfile as string | undefined) ?? 'Dockerfile',
 			context: composeBuild.context as string,
```

Comment:
Is context required?

Reply:
Yes, according to the docs: https://docs.docker.com/compose/compose-file/build/#context-required